adserver/templates/adserver/slack/flight-complete.slack

```diff
@@ -0,0 +1,6 @@
+{% extends django_slack %}
+
+
+{% block text %}
+Flight {{ flight }} for {{ flight.campaign.advertiser }} wrapped up today: {{ flight_url }}
```

Comment:
Should we include any data about it? CTR? Spend? Other active flights?

Reply:
I'm not sure what the most relevant pieces are although those are pretty good. I figure get a v1 out and iterate.